README.md

```diff
@@ -44,10 +44,14 @@ To deploy the app to a RHOAM cluster, you will need to:
 1. Login to the RHOAM cluster using ` oc login ` command
 2. Set the following environment variables:
    ```bash
+   # Mandatory env var. If not set, assuming it is false - deploying for RHMI installation type. 
    export RHOAM=true
+      
+   # This env var must be set if you are deploying web-app wor the multitenant-managed-api installation type. If not set, assuming managed-api installation type. 
+   export SANDBOX=true
+   
    # This env var is optional. Only set it if you want to view the metrics data using the Grafana dashboard
    export GRAFANA_DASHBOARD=true
-
```

Comment:
While this is alright, personal preference is to add an additional step instead, something like:

```
3. Set the following additional variables  for multitenant-managed-api installation type
  ```bash
  # This env var must be set if you are deploying web-app for the multitenant-managed-api installation type. If not set, assuming managed-api installation type. 
   export SANDBOX=true
```
Otherwise I feel it's too easy to just copy and paste the code block without reading the code comments and end up just trying to do a sandbox install on a regular RHOAM cluster, whereas there is a more of a break in an additional step and code block that will make this more obvious :thinking: 

What do you think ?

Reply:
hmm, I had my doubts about a new section - right now we are distinguishing between the RHOAM and the RHMI clusters. It looked weird to have `RHOAM clusters` and `Sandbox clusters` next to each other with copied and pasted sections :thinking: but I don't have any strong opinion on this tho...
Maybe a warning or make an extra subsection with something like `3. Only for Sanbox clusters`? :thinking:

Reply:
Yeah, an extra subsection with `3. Only for Sanbox clusters` sounds good to me :+1:

Reply:
Updated